Makefile

```diff
@@ -0,0 +1,242 @@
+#
+# INTRODUCTION
+#
+# This makefile is designed to help Bisq contributors get up and running
+# as quickly as possible with a local regtest Bisq network deployment,
+# or 'localnet' for short. A localnet is a complete and self-contained
+# "mini Bisq network" suitable for development and end-to-end testing
+# efforts.
+#
+#
+# REQUIREMENTS
+#
+# You'll need the following to proceed:
+#
+#  - Linux, macOS or similar *nix with standard tools like `make`
+#  - bitcoind and bitcoin-cli (`brew install bitcoin` on macOS)
+#  - JDK 10 to build and run Bisq binaries; see
+#    https://www.oracle.com/java/technologies/java-archive-javase10-downloads.html
+#
+#
+# USAGE
+#
+# The following commands (and a couple manual instructions) will get your
+# localnet up and running quickly.
+#
+# STEP 1: Build all Bisq binaries and set up localnet resources. This will
+# take a few minutes the first time through.
+#
+#     $ make
+#
+# Notes:
+#
+#  - When complete, you'll have a number of scripts available in the
+#  root directory. They will be used in the make targets below to start
+#  the various Bisq seed and desktop nodes that will make up your
+#  localnet:
+#
+#     $ ls -1 bisq-*
+#     bisq-desktop
+#     bisq-monitor
+#     bisq-pricenode
+#     bisq-relay
+#     bisq-seednode
+#     bisq-statsnode
+#
+#  - You will see a new 'localnet' directory containing the data dirs
+#  for your regtest Bitcoin and Bisq nodes. Once you've deployed
+#  them in the step below, the directory will look as follows:
+#
+#     $ tree -d -L 1 localnet
+#     localnet
+#     ├── alice
+#     ├── bitcoind
+#     ├── bob
+#     ├── mediator
```

Comment:
I think mediator and seednode folders come after step 2.

Reply:
That's correct, and to this end, I mention in the documentation blurb directly above:

> Once you've deployed [all Bitcoind and Bisq nodes] in the step below, the directory will look as follows: